catalyst/registry/registry.py

```diff
@@ -270,7 +277,7 @@ def get_from_params(
         Returns:
             result of calling ``instantiate_fn(factory, **config)``
         """
-        name = kwargs.pop(self._name_key, None)
+        name = kwargs.pop("name", None)
```

Comment:
how do you think, could we use something like `_name_`, `_key_`, `_target_` here to make it 100% compatible with all possible args/kwargs? I suppose, that `name` could lead to an error if there would we `name` arg if class args/kwargs.